### PR TITLE
feat: retrieve multiple service logs stream at once via cli

### PR DESCRIPTION
**Pull request by tedim52**
## Description
https://www.loom.com/share/aa413be508564af9831a67420d0fb934

## Is this change user facing?
YES

### CODE REVIEW

File: cli/cli/commands/service/logs/logs.go [hunk 1]
```diff
@@ -34,10 +35,12 @@ const (
...
 
 	serviceIdentifierArgKey        = "service"
-	isServiceIdentifierArgOptional = false
-	isServiceIdentifierArgGreedy   = false
+	isServiceIdentifierArgOptional = true // don't need to pass this in if they use the return all services flag
+	isServiceIdentifierArgGreedy   = true
 
 	shouldFollowLogsFlagKey  = "follow"
+	returnAllServiceLogs     = "all-services"
+	allServicesWildcard      = "*"
```
Comment (tedim52) [inline cli/cli/commands/service/logs/logs.go:43]: a user requested a wildcard experience like `kurtosis service logs enclave *` where if they passed in the wildcard all service logs would be returned but you hvae to pass it in like `kurtosis service logs enclave '*'` or else my shell tries to interpret the wildcard , lmk if you have better suggestions

Comment (h4ck3rk3y) [reply cli/cli/commands/service/logs/logs.go:43]: hmm, not great but nothing better immediately comes to mind

Comment (h4ck3rk3y) [reply cli/cli/commands/service/logs/logs.go:43]: maybe a `--all` flag that autocompletes?

**Review by h4ck3rk3y — APPROVED** [review]
beautiful

File: cli/cli/commands/service/logs/logs.go (hunk 1 above)
Comment (tedim52) [reply cli/cli/commands/service/logs/logs.go:43]: `-a` , `-all` is already used for "get all logs of the service" right now, so we'll just rock with the wildcard thing for now

**Pull request merged**